db/versions/0023.yml

```diff
@@ -0,0 +1,43 @@
+version: 23
+migrationScript: 0023-migration.sql
+downgradeScript: 0023-downgrade.sql
+methods:
+  add_github_access_token:
+    description: |-
+      Sets the encrypted access token for user_id_in to
+      encrypted_access_token_in.
```

Comment:
In the future we might render the description somewhere where markdown compatible. We can probably wrap column names with back ticks in the description.

Reply:
Sure I'll add backticks.